src/ufw.yml

```diff
@@ -23,4 +22,12 @@
       loop:
         - "22"       # ssh for Guacamole server (internal to assessment env)
         - "5901"     # vnc for Guacamole (internal to assessment env)
+    - name: Deny incoming traffic to specified TCP ports
+      # These rules may be manually modified by assessors as needed, but we
+      # want to default to a minimal set of allowed ports.
+      community.general.ufw:
+        proto: tcp
+        rule: deny
+        to_port: "{{ item }}"
+      loop:
         - 8000:8999  # Additional listeners, if necessary (external)
```

Comment:
This section can be removed as well.  The default behavior for incoming traffic is `deny`.

Reply:
In https://github.com/cisagov/cool-system-internal/issues/80, it sounds like @m1j09830 wants these `deny` rules to be explicitly defined, but I'll let him confirm this.  I'm happy to remove this if it's not needed.

Reply:
@dav3r yes sir that is the idea. If it's explicitly defined then the teams will have a greater likelihood of identifying when something isn't working due to a firewall rule vs a technical issue on their end. For example if they are hosting payloads and unable to hit the payload site but don't see a firewall rule they may assume something is wrong with the hosting server and spend cycles going down an unnecessary rabbit hole. Just trying to prevent as much confusion upfront as possible.

Reply:
Comment added in https://github.com/cisagov/kali-packer/pull/92/commits/789f695e129fa708b54ac84d85e2986de3060319 - please let me know if that works for you.

Reply:
The comment is great.  Are the ports 8000-8999 still relevant, or do those need to be changed to 5000-5999?

Reply:
If I'm understanding @m1j09830 correctly, it sounds like those ports are still relevant here because they may be used by assessors on an as-needed basis.  @m1j09830, please correct me if that is wrong.

Reply:
@dav3r I submitted a request to the NOM team per our discussion to have this port range removed from the TeamServers. These ports are only relevant on the Kali Images once NOM makes the requested changes. That said you can feel free to remove them (8000-8999).

@jsf9k we will need to include the port range 5000-5999 as this range is used for internal comms between hosts.

Reply:
> This should be in a default ALLOW state though since they are for internal comms only and have no risk of being exposed externally.

I wouldn't say that there is **no risk** of those ports being exposed externally.  There is still the possibility of someone misconfiguring an environment and opening up some/all of those ports publicly, but the risk of that is fairly low with our current deployment process.

Reply:
Please see https://github.com/cisagov/kali-packer/pull/92/commits/0a537c9fadba4e99d513c3e006e9d3e5dd835f57 and https://github.com/cisagov/kali-packer/pull/92/commits/1423aefd504df01947b3958d740cd31559f18c3b for the requested updates and let me know if anything looks awry now.

Reply:
Looks good to me!